website/www/site/content/en/community/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Community Beam"
+aliases:
+  - /use/issue-tracking/
+  - /use/mailing-lists/
+  - /get-started/support/
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Welcome to the Beam Community!
+
+Beam is a tool created by community for community. We tirelessly work to make it better and you can do it too!
+If you are a data or software developer this is a place for you.
+
+{{< community/list_with_icons community_list >}}
```

Comment:
This copy also wasn't updated in the last commit. Is this the final copy, or are we still waiting on it?

Reply:
From what I know it's final copy

Reply:
Hm. I don't see this page in the TDD

Reply:
Updated

Reply:
Thanks!